src/managed_data/api.py

```diff
@@ -13,6 +14,86 @@
 from deeporigin.utils import PREFIXES
 
 
+@beartype
+def make_database_rows(
+    database_id: str,
+    n_rows: int = 1,
+    client: Optional[Client] = None,
+) -> dict:
+    """Makes one or several new row(s) in a Database table
+
+    This wraps the `EnsureRows` endpoint and sends a payload
+    designed to create new row(s) in a database table.
+
+
+    Args:
+        database_id: ID or Human ID of the database
+
+    Returns:
+        A dictionary that conforms to a EnsureRowsResponse
+    """
+
+    if n_rows < 1:
+        raise DeepOriginException(
+            f"n_rows must be at least 1. However, n_rows was {n_rows}"
+        )
+
+    data = dict(
+        databaseId=database_id,
+        rows=[{"row": {}} for _ in range(n_rows)],
+    )
+
+    return _api.ensure_rows(data, client=client)
```

Comment:
this is now a high level function, and calls a low-level function 

this restores the pattern where high level functions don't know anything about requests, or about the client